---
description: Literature and Book Recommendations
---

# Machine learning & AI

## 0. Courses & Books 

* **Courses:** 
  * Machine Learning \(CS229\) -- Andrew Ng : [coursera](https://www.coursera.org/learn/machine-learning)
* **Books**
  * **Probabilistic Models** \(HMM & SCFG\): 《[Biological Sequence Analysis](http://www.amazon.com/Biological-Sequence-Analysis-Probabilistic-Proteins/dp/0521629713/):Probabilistic Models of Proteins and Nucleic Acids》 \([English](http://www.amazon.com/Biological-Sequence-Analysis-Probabilistic-Proteins/dp/0521629713) \| [中文](http://www.amazon.cn/dp/B003ZUIRZ2)\) by _Richard Durbin, Sean R. Eddy, Anders Krogh, Graeme Mitchison_
  * **Macine Leanring :**《机器学习》-- 周志华 \([link](https://book.douban.com/subject/26708119/)\)
* **More**: [更多书籍课程推荐](https://lulab1.gitbook.io/training/appendix/appendix-i.keep-learning)

## 1. Startup of Literature Reading

* **Deep Learning**: 2015 Nature Biotech. - DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning
* **Cancer Locator**:  2017 Genome Biology - CancerLocator: non-invasive cancerdiagnosis and tissue-of-origin predictionusing methylation profiles of cell-free DNA
* **Batch correction**: 2018 Nature Biotech. - Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors

[PDFs](https://cloud.tsinghua.edu.cn/d/928f3f4a8c8d4ab8b8ad/?p=/Literature%20shared%20by%20John/Recommendation%20for%20Startup/AI%20%26%20Machine%20Learning&mode=list)

## 2. Deep Learning on RNA

* **AS of RNA**: 2019 Cell - Predicting Splicing from Primary Sequence with Deep Learning
* **AS of RNA \(DARTS**\): 2019 Nature Methods - Deep-learning augmented RNA-seq analysis of transcript splicing
* **APA of RNA**: 2019 Cell - A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation
* **RNA/DNA-Protein Binding \(DeepBind\)**: 2015 Nature Biotech. - DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning
* **RNA secondary structure**: see [below](ai.md#3-1-rna-secondary-structure-prediction)
* **Cancer prediction using exRNA-seq**: see [below](ai.md#4-5-transfer-learning)

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FDeep%20Learning%20on%20RNA&mode=list)

## 3. RNA Structure

### 3.1 RNA Secondary Structure Prediction

1. **Dynamic Programming Method \(Energy Model\):** RNAstructure/RNAfold
2. **SCFG Method:** Rfam
3. **Deep Learning Method**: 2019 Nature Commn. - SPOT-RNA: RNA secondary structure prediction using an ensemble of two-dimensional deep neural networks and transfer learning  

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FRNA%20secondary%20structure%20prediction&mode=list)

### 3.2 Structural Motif Finder

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FRNA%20Structural%20Motif&mode=list)

## 4. Comp. Methods for Liquid Biopsy

### 4.1 Normalization and Batch Correction

* **Methods for Single Cell RNA-seq**: imputation, normalization, clustering, etc
  * Heterogeneity and Normalization
  * Batch effect and confounder
* **Recommendations**:
  * 2017 - Nature Methods - Normalizing single-cell RNA sequencing data: challenges and opportunities
  * 2019 - Nature Methods - A discriminative learning approach to differential expression analysis for single-cell RNA-seq

> [PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20single%20cell&mode=list)
>
> [Tutorial](https://lulab1.gitbook.io/training/part-iii.-case-studies/case-study-1.exrna-seq/1.4.normalization-issues)

### 4.2  Feature Selection Method

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20Feature%20Selection&mode=list)

### 4.3 Network Approach

* 2018 Nature COMMN. - Pathway based subnetworks enable cross-disease biomarker discovery

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20Feature%20Selection&mode=list)

### 4.4 Tumor Location Method

* [Methods for Metagenomics](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20metagenomics&mode=list): tumor locator \(tissue match\), ranking, etc
* [Methods for Tumor Location](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20locator&mode=list): tumor locator \(tissue match\), etc

### 4.5 Transfer Learning

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FTransfer%20Learning&mode=list)



